interview.md

```diff
@@ -5,47 +5,93 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+    The user is someone who needs to manage heat pumps in different homes.
+    A language might be a good way to manage individual parameters of different
+    heat pumps.
+
 ### What's the need?
 
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+    The need is met as many different parameters need to be taken into account
+    before installing a heat pump. I would be helping people like my brother who
+    need to install heat pumps to be as energy efficient as possible. Their
+    experience currently is that they need to take different temperatures, different
+    readings from data not publicly available, and analyze this data to determine
+    when best to have heatpumps running. This experience might be easier if
+    a DSL was used to input these parameters and output automatically the most
+    efficient and best parameters.
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+    A DSL is appropriate because it is specific to a domain and would
+    eliminate human error from analyzing data. It would also speed up
+    this process of gathering data/analyzing it.
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+    I like the idea of saving energy/cost and that having an impact
+    on potentially millions of people! I came up with it after
+    speaking to my brother, who is an energy consultant. I think.
+    I'm not sure what his official job title is but his job is to
+    come up with ways for businesses and homes to use less energy
+    and to save money by using renewable energy sources, and he
+    thought that step in the process could be automated by a
+    DSL.
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+    Automated parameter setup of heatpumps.
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+    The user could interact with the language by inputting in
+    specific parameters they have access to. This could look like
+    a terminal window or a text file. This is the right way to
+    interact with the domain as the information applicable could
+    vary from building to building.
+
 ### Operation (semantics)
 
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+    When a program runs the right parameters for these heatpumps
+    could be output either as a csv file or a text file. The program
+    could interact with the user through a terminal/command prompt, which
+    could output an error in the case that syntax errors occur.
+
```

Comment:
we gotta be careful, I feel like errrors here could be catastrophic. Its always scarier working with management systems like this